src/lib/expansion/expansion-animations.ts

```diff
@@ -20,6 +20,11 @@ import {
 /** Time and timing curve for expansion panel animations. */
 export const EXPANSION_PANEL_ANIMATION_TIMING = '225ms cubic-bezier(0.4,0.0,0.2,1)';
 
+// Parameters for `expansionHeight` animation states. Need to be pulled out
+// into constants, in order to prevent Closure from renaming them.
+const EXPANDED_HEIGHT_PARAMS = {params: {expandedHeight: '64px'}};
+const COLLAPSED_HEIGHT_PARAMS = {params: {collapsedHeight: '48px'}};
```

Comment:
I don't follow why moving these into constants would prevent Closure compiler from renaming the properties.

Reply:
That's at least what I got from our discussion last time and the issue description, but maybe I misunderstood.

Reply:
I don't remember the exact discussion, but I think what was probably referenced was doing this:
```ts
declare const collapsedHeight: string;
declare const expandedHeight: string;
```

Anything with the same name as a global `declare const` will be ignored by Closure Compiler. We have this just set locally in Google now

Reply:
I've changed it. Is having them as `declare const` enough or do they have to be actual constants?